packages/app-project/src/screens/ProjectHomePage/components/Hero/components/OrganizationLink/OrganizationLink.js

```diff
@@ -0,0 +1,38 @@
+import { Anchor, Box } from 'grommet'
+import { useTranslation } from 'next-i18next'
+import { string } from 'prop-types'
+import { SpacedText } from '@zooniverse/react-components'
+
+function OrganizationLink({
+  slug = '',
+  title = ''
+}) {
+  const { t } = useTranslation('components')
+
+  return (
+    <Box
+      alignContent='start'
+      wrap={true}
+    >
+      <SpacedText>
+        {t('ProjectHeader.organization')}
+      </SpacedText>
+      <Anchor
+        href={`https://www.zooniverse.org/organizations/${slug}`}
```

Comment:
Needs to be updated to reflect domain per ENV

Reply:
```suggestion
        href={`/organizations/${slug}`}
```